src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -1460,7 +1587,7 @@ public function guessMimeType(string $path): ?string
         'text/x-lua' => ['lua'],
         'text/x-lyx' => ['lyx'],
         'text/x-makefile' => ['mk', 'mak'],
-        'text/x-markdown' => ['md', 'mkd', 'markdown'],
+        'text/x-markdown' => ['mkd', 'md', 'markdown'],
```

Comment:
if the 1st is the most used, then `md` should be first?
does the order mean something in the source db?

Reply:
For reference, the source DB uses this preference for merging the multiple sources:

```
// source preference (least -> most)
var preference = ['nginx', 'apache', undefined, 'iana']
```